vulnerable-code-new.js

```
@@ -0,0 +1,87 @@
+    return database.query(query);
+}
+
+// 2. 매우 위험한 eval() 사용
+function executeUserCode(userInput) {
+    console.log("About to execute user code:", userInput);
+    return eval(userInput); // 극도로 위험!
+}
+
+// 3. 하드코딩된 민감한 정보
+const API_KEY = "sk-1234567890abcdef";
+const DATABASE_PASSWORD = "admin123";
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)